db/migrate/20150828182207_generate_tokens_for_existing_users.rb

```diff
@@ -0,0 +1,7 @@
+class GenerateTokensForExistingUsers < ActiveRecord::Migration
+  def change
+    User.all.each do |user|
```

Comment:
Rather than using schema migration for data migration - I would recommend to check custom rake tasks, that you can run in any environment. http://guides.rubyonrails.org/command_line.html#custom-rake-tasks